app/enterprise/changelog.md

```diff
@@ -4,6 +4,127 @@ no_search: true
+causes the plugin to ensure a valid Content-Length header exists before reading the request body.
+- The [Serverless Functions](https://docs.konghq.com/hub/kong-inc/serverless-functions/) 
+(`serverless-functions`) plugin introduces a sandboxing capability, is *enabled* by default, 
+and where only Kong PDK, OpenResty `ngx` APIs, and Lua standard libraries are allowed.
+
+#### Configuration
+- `client_max_body_size` and `client_body_buffer_size`, that previously
+  hardcoded to 10m, are now configurable through `nginx_admin_client_max_body_size` 
+  and `nginx_admin_client_body_buffer_size`.
+- Kong-generated SSL private keys now have `600` file system permission.
+- Properties `ssl_cert`, `ssl_cert_key`, `admin_ssl_cert`,
+  `admin_ssl_cert_key`, `status_ssl_cert`, and `status_ssl_cert_key`
+  is now an array: previously, only an RSA certificate was generated
+  by default; with this change, an ECDSA is also generated. On
+  intermediate and modern cipher suites, the ECDSA certificate is set
+  as the default fallback certificate; on old cipher suite, the RSA
+  certificate remains as the default. On custom certificates, the first
+  certificate specified in the array is used.
+- Kong now runs as a `kong` user if it exists; if user does not exist
+  in the system, the `nobody` user is used, as before.
+  
+### Dependencies
+- Bumped `kong-plugin-serverless-functions` from 1.0 to 2.1.
+- Bumped `lua-resty-dns-client` from 5.1.0 to 5.2.0.
+- Bumped `lua-resty-healthcheck` from 1.3.0 to 1.4.0.
+- Bumped `OpenSSL` from 1.1.1h to 1.1.1i.
+- Bumped `kong-plugin-zipkin` from 1.1 to 1.2.
+- Bumped `kong-plugin-request-transformer` from 1.2 to 1.3.
+
+### Fixes
+
+#### Core
+- Fixed issue where a Go plugin would fail to read `kong.ctx.shared` values set by LUA plugins.
+- Properly trigger `dao:delete_by:post` hook.
+- Fixed issue where a route that supports both http and https (and has a hosts and snis 
+match criteria) would fail to proxy http requests, as it does not contain an SNI.
+- Fixed issue where a `nil` request context would lead to errors `attempt to index local 'ctx'`
+being shown in the logs.
+- Reduced the number of needed timers to active health check upstreams and to resolve hosts.
+- Schemas for full-schema validations are correctly cached now, avoiding memory
+  leaks when reloading declarative configurations. 
+- The schema for the upstream entities now limits the highest configurable
+  number of successes and failures to 255, respecting the limits imposed by
+  `lua-resty-healthcheck`. 
+- Certificates for database connections now are loaded in the right order
+  avoiding failures to connect to Postgres databases.
+
+#### CLI
+- Fixed issue where `kong reload -c <config>` would fail.
+- Fixed issue where the Kong configuration cache would get corrupted.
+
+#### Developer Portal
+- Fixed issue when applying permissions to developers using the Application Registration feature.
```

Comment:
How would a user notice this issue / what error(s)/failure(s) would they see?

Reply:
> How would a user notice this issue / what error(s)/failure(s) would they see?

I'm not familiar with this issue. @HenriPro do you know anything about this?

Reply:
Will fix this in GA.